src/utils/input-utils.ts

```diff
@@ -278,9 +278,9 @@ const integerFieldValidation = {
   message: ERROR_MESSAGES.INVALID_INTEGER
 };
 
-const numericFieldValidation = {
-  value: dataFormats.regexp.float,
-  message: ERROR_MESSAGES.INVALID_NUMERIC
+const numericFieldValidation = (value: any) => {
+  if (!value) return;
+  return !isNaN(parseFloat(value)) && isFinite(value) || ERROR_MESSAGES.INVALID_NUMERIC;
```

Comment:
This doesn't properly handle leading/trailing whitespace. So for example `123123.23  ` and `   123` are treated like valid numbers. 

ermrest is actually properly ignoring the whitespace when we send these values, but we most probably should just invalidate such values, right?

If we want to disallow any white space we can just do `/\s/g.test(s)` or if we want to only disallow leading/trailing we could `.trim()` and see if they match or not .